### DISCUSSION TITLE
[SvelteKit] Stay logged in

### DISCUSSION THREAD

**Discussion by tinsever** [pocketbase/pocketbase#5318] [Q&A] [answered]
Hello everyone,

what would be the best way to **stay** logged in in Svelte? I cant "just" set a session cookie with the id.

**Comment by tinsever** [comment, marked as answer]
Ah, I forgot a part in my tutorial! Sorry for the inconvinience!

**Reply by ganigeorgiev** [reply]
_Side-note: make sure to check https://github.com/pocketbase/pocketbase/discussions/5313._

**Reply by tinsever** [reply]
Thank you!
